comp/core/agenttelemetry/impl/config.go

```diff
@@ -200,13 +200,8 @@ var defaultProfiles = `
         - name: transactions.input_count
         - name: transactions.requeued
         - name: transactions.retries
-        - name: dogstatsd.udp_packets
-          aggregate_tags:
-            - state
-        - name: dogstatsd.uds_packets
-          aggregate_tags:
-            - transport
-            - state
+        - name: dogstatsd.udp_packets_bytes
+        - name: dogstatsd.uds_packets_bytes
```

Comment:
This change should be accommodated with the corresponding change on actual metric (at least currently). Second question you are not interested in tag partitioning (less values to emit the better of course).

Reply:
Oh wait a seconds, are there actual metrics like that? Let me check.

Reply:
Ok, never mind my comment